src/apps/backend/modules/config/config.py

```diff
@@ -0,0 +1,157 @@
+import os
+from pathlib import Path
+from typing import Any, Optional
+import yaml
+
+
+def get_parent_directory(directory: str, levels: int) -> Path:
+    parent_dir = Path(directory)
+    for _ in range(levels):
+        parent_dir = parent_dir.parent
+    return parent_dir
+
+
+class Config:
+    config_dict: dict[str, Any]
+    config_path: Path = get_parent_directory(__file__, 6) / "config"
+
+    @staticmethod
+    def read(filename: str) -> dict[str, Any]:
+        file_path = Config.config_path / filename
+        yaml_content: dict[str, Any] = {}
+        try:
+            with open(file_path, "r", encoding="utf-8") as file:
+                yaml_content = yaml.safe_load(file)
+        except FileNotFoundError as e:
+            print(e)
+        return yaml_content
+
+    @staticmethod
+    def load_config() -> None:
```

Comment:
1. If this is the only public method, move to top of the class.
2. Mark all internal methods to start with `_`
3. Consider improving readability of the code overall from code organization, naming and others